spec/server.spec.js

```diff
@@ -20,3 +23,43 @@ describe("데이터베이스", () => {
     expect(isConnected).toBe(true);
   });
 });
+
+describe("/api/performance-comparison", () => {
+  it("함수 이외의 타입을 코드 에디터에 입력한 경우 상태코드 400을 반환해야 한다.", async () => {
+    const { req } = createMocks({
+      method: "POST",
+      body: {
+        functionCode: "const a = 1",
+        functionArguments: ["a", "b", "c"],
+      },
+    });
```

Comment:
```suggestion
  describe("/api/performance-comparison", () => {
  let req;

  beforeEach(() => {
    req = createMocks({
      method: "POST",
      body: {},
    });
  });

  it("함수 이외의 타입을 코드 에디터에 입력한 경우 상태코드 400을 반환해야 한다.", async () => {
    req.body = {
      functionCode: "const a = 1",
      functionArguments: ["a", "b", "c"],
    };
```
테스트 함수 내에 공통적으로 사용되는 부분은 `beforeach`를 사용하면 더 가독성이 좋을 것 같아요!

`beforeach`로 `req`의 공통 부분을 `it`에 각각 전달해주고 `it`에서는 `req.body`만 정의해주는 방식으로 해보는건 어떨까요?

Reply:
`beforeEach` 훅을 사용하여 중복을 제거하니 훨씬 가독성이 좋아보입니다!

코드에 적용하다보니 `beforeEach`훅을 사용하지 않고 `describe` 블락문에 아래 코드처럼 req 변수를 선언하고 사용하는게 더 가독성이 좋다고 판단되는데 리뷰어분들 의견은 어떤가요?

```
const req = createMocks({
      method: "POST",
    });
```

Reply:
`beforeEach` 나 `describe 블락문에 아래 코드처럼 req 변수를 선언`은 **반복되는 코드를 줄인다는 부분에서 목적**은 같지만 어떻게 해당 부분을 어떻게 처리할 것 인지가 의견이 달라지는 부분이라고 생각합니다.

개인적으로는  단순히 `req` 객체를 정의하는 부분이 반복되는 것 이라고 생각해  현재 상황에서는 `beforeEach`를 사용하는 것 보다는 `describe 블락문에 아래 코드처럼 req 변수를 선언` 하여 `req.body` 만 테스트마다 재정의하여 사용하는 것이 괜찮다고 생각합니다.

만약 이후 반복되는 부분이 조금 더 **안정성이 있어야하는 부분** 이거나 **반복되는 코드가 복잡**하다면 `
beforeEach`를 사용하여 개선하는 것이  좋다고 생각합니다🙇🏻

Reply:
알겠습니다. 의견 반영하여 수정하겠습니다!